ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/linear_optimisation/fillers/CoreProblemFiller.java

```diff
@@ -56,10 +69,39 @@ public void fill(RaoData raoData, LinearProblem linearProblem) {
 
     @Override
     public void update(RaoData raoData, LinearProblem linearProblem) {
+        // chose range actions to use
+        computeAvailableRangeActions(raoData);
```

Comment:
maybe we can keep the ones of the fill() to avoid strange loops in the iterations ?

Reply:
okay